src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/LSPRazorProjectHost.cs

```diff
@@ -1,137 +1 @@
-            _languageClients = languageClients;
-        }
-
-        private List<Lazy<ILanguageClient, ILanguageClientMetadata>> ApplicableLanguageClients { get; set; }
-
-        public async Task LoadAsync()
-        {
-            await _joinableTaskFactory.SwitchToMainThreadAsync();
-
-            if (!_featureDetector.IsLSPEditorFeatureEnabled())
-            {
-                return;
-            }
-
-            ApplicableLanguageClients = GetApplicableClients(_languageClients);
-
-            foreach (var client in ApplicableLanguageClients)
-            {
-                _languageClientBroker.Value.LoadAsync(client.Metadata, client.Value).Forget();
-            }
-        }
-
-        public Task UnloadAsync()
-        {
-            return Task.CompletedTask;
-        }
-
-        private static List<Lazy<ILanguageClient, ILanguageClientMetadata>> GetApplicableClients(IEnumerable<Lazy<ILanguageClient, IDictionary<string, object>>> languageClients)
-        {
-            var applicableClients = new List<Lazy<ILanguageClient, ILanguageClientMetadata>>();
-            foreach (var client in languageClients)
-            {
-                if (!client.Metadata.TryGetValue(nameof(ILanguageClientMetadata.ContentTypes), out var contentTypeValue) ||
-                    !(contentTypeValue is IEnumerable<string> contentTypes) ||
-                    !contentTypes.Intersect(_applicableContentTypes).Any())
-                {
-                    continue;
-                }
-
-                string clientName = null;
-                if (client.Metadata.TryGetValue(nameof(ILanguageClientMetadata.ClientName), out var clientNameValue))
-                {
-                    clientName = clientNameValue.ToString();
-                }
-
-                var disableUserExperience = false;
-                if (client.Metadata.TryGetValue("DisableUserExperience", out var disableUserExperienceValue))
-                {
-                    disableUserExperience = (bool)disableUserExperienceValue;
-                }
-
-                applicableClients.Add(new Lazy<ILanguageClient, ILanguageClientMetadata>(
-                    () => { return client.Value; },
-                    new LanguageClientMetadata(clientName, contentTypes, disableUserExperience)));
-            }
-
-            return applicableClients;
-        }
-
-        private class LanguageClientMetadata : ILanguageClientMetadata, IIsUserExperienceDisabledMetadata
-        {
-            public LanguageClientMetadata(string clientName, IEnumerable<string> contentTypes, bool isUserExperienceDisabled)
-            {
-                ClientName = clientName;
-                ContentTypes = contentTypes;
-                IsUserExperienceDisabled = isUserExperienceDisabled;
-            }
-
-            public string ClientName { get; }
-
-            public IEnumerable<string> ContentTypes { get; }
-
-            public bool IsUserExperienceDisabled { get; }
-        }
-    }
-}
+﻿
```

Comment:
was this file supposed to be deleted instead of left as an empty file?

Reply:
ya wth, I wonder how that happened ❤️